shared/tests/utility/math/docs/README.txt

```diff
@@ -0,0 +1,14 @@
+The files in this folder contain the same input data used in TestCoordinates.cpp along with some matlab scripts that perform the same calculations found in coordinates.h
+The output from matlab is used to validate the coordinates.h calculations in the NUBOTS codebase.
+Test input data can be found in both matlab and c++ formats. The matlab format data is passed into the functions starting with calc_****.m to produce the output data.
+Test output data provided is in a cpp format.
+
+NOTE: During initial testing when values at the extremes of the data types domain, DBL_MAX and DBL_MIN, were found to cause inf return values when passed to the functions
+being tested. While normally some measures would have been put in place to handle this, it was determined that the performance impact would be to great, and that these values won't be
+possible during normal operation.
```

Comment:
Noting here as well that it's not that the values aren't possible, it's more that we're assuming they won't happen.